### DISCUSSION TITLE
FaunaDB comparison?

### DISCUSSION THREAD

**Discussion by tommyc38** [supabase/supabase#910] [Questions] [answered]
FaunaDB did a comparison on Postgres.  I was wondering if someone from the Supa team could do one for Fauna vs Supa?  Fauna claims they are a gen 2 or 3 database while Postgres is a gen 1 (From Summary in link below).  

https://docs.fauna.com/fauna/current/comparisons/compare-faunadb-vs-postgres.html

**Comment by silentworks** [comment, marked as answer]
Comparisons are great but they are also always biased, I don't think there is a need for a comparison as supabase does what it does really well and from what I have been told about faunadb it also does what it does really well too. I think the main difference between the two is you can move away from using supabase infra if your app becomes that big at any point or if you are unhappy with something since its just Postgres under the hood. I don't think this is possible with faunadb. Choose the one that will work best for your project/team/company and go with that, forget about comparisons as they are all biased.

**Reply by tommyc38** [reply]
Thanks Steve.  I agree with everything you said 🤙🏻
